### DISCUSSION TITLE
cmdlet to invoke service-now endpoint

### DISCUSSION THREAD

**Discussion by cmblackburn** [Snow-Shell/servicenow-powershell#269] [Ideas]
Any chance in adding a cmdlet to invoke one of service-now methods, it would be a great addition if there was a way to use the built-in API's in service-now with this module.

**Comment by gdbarron** [comment]
Hi @cmblackburn.  Can you be more specific please, which APIs are you referring to?  What are you thinking/hoping this would look like from a user perspective?

**Reply by cmblackburn** [reply]
Good morning @gdbarron!  I hope you're doing well!  The use case I'm currently doing is for the endpoint for getting the latest on-call, but there are many end-points that service-now provides, https://my.service-now.com/api/now/on_call_rota/whoisoncall.

I could see it being invoke-servicenowmethod -method whoisoncall -filter @{"groupid":"adlsjfa;sdjf;afsj"}

Thanks for looking into this!
